sdk/monitor/azure-monitor-opentelemetry-exporter/src/main/java/com/azure/monitor/opentelemetry/exporter/AzureMonitorExporterOptions.java

```diff
@@ -285,31 +272,13 @@ public AzureMonitorExporterBuilder credential(TokenCredential credential) {
         return this;
     }
 
-    /**
-     * Creates an Azure Monitor span exporter based on the options set in the builder. This
-     * exporter is an implementation of OpenTelemetry {@link SpanExporter}.
-     *
-     * @param configProperties The OpenTelemetry configuration properties.
-     * @return An instance of {@link SpanExporter}.
-     * @throws NullPointerException if the connection string is not set on this builder or if the
-     * environment variable "APPLICATIONINSIGHTS_CONNECTION_STRING" is not set.
-     */
-    public SpanExporter buildSpanExporter(ConfigProperties configProperties) {
+    SpanExporter buildSpanExporter(ConfigProperties configProperties) {
```

Comment:
may make sense to move the `build*` methods out of the `*Options` class, but we can do that later now that these are package-private (I like that the current PR is minimal changes)